lib/puppet/indirector/certificate_revocation_list/rest.rb

```diff
@@ -7,4 +7,16 @@ class Puppet::SSL::CertificateRevocationList::Rest < Puppet::Indirector::REST
   use_server_setting(:ca_server)
   use_port_setting(:ca_port)
   use_srv_service(:ca)
+
+  def find(request)
```

Comment:
We have to have this guard clause because we might need to make network operations in order to configure SSL, which causes infinite recursion. With the file terminus the file is there or it's not, and we won't recurse.